### ISSUE TITLE
Worth revising installation instructions for MacOS?

### ISSUE THREAD

**Issue by palvarezc** [palvarezc/cavendish-particle-tracks#84]
Using a conda environment with python3.11 in MacOS and trying to install as suggested using 'python -m pip install -e ".[testing]" fails for me, but seems to work on the ci machines, not sure why. [This thread](https://github.com/altendky/pyqt-tools/issues/98) seems to point to pyqt5 in python>3.9 being related to the issue. 

Managed to make it work installing napari with conda first (as [napari recommends](https://napari.org/stable/tutorials/fundamentals/installation.html#choosing-a-different-qt-backend) 🙃)
```bash
conda install -c conda-forge napari pyqt
python -m pip install -e ".[testing]"
```

I dont know how `tox` handles this...

**Comment by Joseph-Garvey** [comment]
Is it an Arm or X86 Mac?

**Comment by palvarezc** [comment]
> Is it an Arm or X86 Mac?

Mine is X86

**Comment by samcunliffe** [comment]
Solved via [501e553](https://github.com/palvarezc/cavendish-particle-tracks/commit/501e553aadfc6855aa273db34732858b85b0e8b9)
